src/constants.cpp

```diff
@@ -1660,6 +1660,17 @@ const char *attributes[] =
     "Reaction"
   };
 
+const char *short_attributes[] =
```

Comment:
Used to confirm which attribute the imm wants affected by the attribute spells in spellset.